.github/workflows/build.yml

```diff
@@ -221,6 +229,8 @@ jobs:
           python-version: 3.7
       - name: Install system dependencies
         run: brew install ffmpeg jpeg-turbo
+      - name: Install utils
```

Comment:
Does this mean mmcv need one more dependency after this PR?

Reply:
Yes

Reply:
then should we update requirements?

Reply:
The problem is that, this pkg is used in the setup.py script, which will be run before requirements are parsed.

So the current code works like this:

* if psutil is installed prior compiling mmcv, then it will use (num_of_cpu - 1) threads
* otherwise, use the default value 4